studio/components/interfaces/Database/Hooks/EditHookPanel.tsx

```diff
@@ -81,7 +86,7 @@ const EditHookPanel = ({ visible, selectedHook, onClose }: EditHookPanelProps) =
     })
   const isSubmitting = isCreatingDatabaseTrigger || isUpdatingDatabaseTrigger
 
-  const tables = meta.tables.list().sort((a, b) => (a.schema > b.schema ? 0 : -1))
+  const tables = data?.sort((a, b) => (a.schema > b.schema ? 0 : -1)) ?? []
```

Comment:
You're sorting in-place here. Will this mess up the `react-query` cache?

Reply:
good call - probably good to put it in the `select` param of the RQ?

Reply:
I'm not sure if react query handles this or not – so fair call, best to play it safe. I've added a copy and sort inside a `useMemo` here instead